go.mod

```diff
@@ -90,6 +90,8 @@ require (
 	github.com/quasoft/websspi v1.1.2
 	github.com/redis/go-redis/v9 v9.4.0
 	github.com/robfig/cron/v3 v3.0.1
+	github.com/russellhaering/gosaml2 v0.9.1
```

Comment:
I found a more popular package([crewjam/saml](https://github.com/crewjam/saml)) that we can use. It looks like [russellhaering/gosaml2](https://github.com/russellhaering/gosaml2) only supports HTTP POST binding, but [crewjam/saml](https://github.com/crewjam/saml) supports both HTTP Redirect binding and HTTP POST binding.

Reply:
if you make a pull against my feature branch we can test that lib :)

Reply:
PS: it's no simple drop-in replacement :/

Reply:
@Zettat123 the pull is not originally from me but was just resubmited for review.

so @techknowlogick or @jackHay22 might know more about why thy have choosen the lib?

Reply:
@Zettat123 I'll added it to the followups is that ok or should we wait for the others to respond?

Reply:
@techknowlogick I noticed that initially you added the SAML auth feature. Could you please take a look at the [crewjam/saml](https://github.com/crewjam/saml) package and evaluate if we can use it?

Reply:
They both look to have similar amount of activity, although having additional options would be nice. I chose the one above as it's the one I've used previously and have experience with, but perhaps crewjam may be better to use if it supports additional items.

@jackHay22 any thoughts?

Reply:
I don't have a strong opinion here; if the additional features are useful this would be a good time to make the switch.

Reply:
@Zettat123 It looks like gosaml2 supports [HTTP redirect binding](https://github.com/search?q=repo%3Arussellhaering%2Fgosaml2%20BindingHttpRedirect&type=code). Let me know if I'm missing something.

Reply:
Found an [issue](https://github.com/russellhaering/gosaml2/issues/98) related to the Redirect binding. I haven't looked into this issue, but it looks like there is a workaround. But I think this issue not being solved for a long time means that the project is not very active.

Reply:
I don't disagree but I think the `crewjam/saml` repo also has signs of inactivity.

Reply:
I think at some point a saml lib is just completed and only needs version bumps if it  has dependencys?

so in this regards if there are no issues open anymore I'm fine with a inactive lib.

here both repos still have open issues :/